app/assets/javascripts/listings/templates/property-card/_rental-stats-row.html.slim

```diff
@@ -1,15 +1,17 @@
 tr ng-repeat-start="unit_summary in ::summary" ng-if="::unit_summary.minMonthlyRent || unit_summary.minPercentIncome"
-  td.is-subtitled data-th="{{'t.unit_type' | translate}}"
+  td.is-subtitled data-th="{{'t.units' | translate}}"
```

Comment:
should we rename these files to -desktop just as a hint that there's another file out there?